### ISSUE TITLE
benchmarks?

### ISSUE THREAD

**Issue by beckermr** [nils-braun/dask-sql#139]
I know they are mostly useless, but they still might add some value. Have any been done?

**Comment by nils-braun** [comment]
A good question nevertheless!
So far not from my side. There are some [benchmarks](https://blog.blazingdb.com/data-lake-to-ai-blazingsql-rapids-initial-benchmark-aa753031ac8b) on blazingSQL if you are interested, but none from dask-sql (that I am aware of).
What would you be interested in?
* dask-sql vs. dask?
* scaling behaviour with #workers and data size?
* dask-sql vs. blazingSQL vs. presto vs. impala vs. spark vs. ...
* anything else?
* dask-sql over time?

I think doing the benchmark is just a matter of spending some cloud-resources and time, for me the problem is more finding a reasonable setup (how many workers? CPUs? RAM?) and a reasonable problem (data + query). I would be happy to hear your thoughts!

**Comment by beckermr** [comment]
> dask-sql vs. blazingSQL vs. presto vs. impala vs. spark vs. ...

Yes, this is what I had in mind. I'd exclude anything that uses a GPU and focus on comparable tools/ideas like spark, presto, and impala.

There is always this set of benchmarks: https://amplab.cs.berkeley.edu/benchmark/

However, they exclude using file formats like parquet. Versions with parquet might be good?
